src/main/kotlin/blackjack/view/InputView.kt

```diff
@@ -13,4 +15,12 @@ object InputView {
         }
         return false
     }
+
+    fun askBettingAmount(name: String): Int {
+        println("${name}의 베팅 금액은?")
+        val amount = readln().toIntOrNull()
+        requireNotNull(amount) { "베팅 금액은 숫자로 입력해야 합니다." }
```

Comment:
```suggestion
        val amount = readln().toIntOrNull() ?: throw IllegalArgumentException("~~~")
```
하나의 라인으로도 표현할 수 있을 것 같습니다!